templates/openwisp2/settings.py

```diff
@@ -156,7 +158,6 @@
 
 # Password validation
 # https://docs.djangoproject.com/en/1.9/ref/settings/#auth-password-validators
-
```

Comment:
never remove lines that are not related to your patch and hence is not necessary to remove them